brightray/browser/browser_main_parts.cc

```diff
@@ -89,6 +89,18 @@ void OverrideLinuxAppDataPath() {
   PathService::Override(DIR_APP_DATA, path);
 }
 
+// void OverrideWinAppLogsPath() {
+//   base::FilePath path;
+//   // should be in c:\program files\myapp\logs ?
```

Comment:
The logs location should be in `%LOCALAPPDATA%` or `%APPDATA` so we don't need admin permissions to write the file